install-panel.sh

```diff
@@ -744,118 +754,59 @@ function configure_nginx {
       ln -s /etc/nginx/sites-available/pterodactyl.conf /etc/nginx/sites-enabled/pterodactyl.conf
   fi
 
-  # restart nginx
-  if [ "$CONFIGURE_LETSENCRYPT" == true ] || { [ "$CONFIGURE_LETSENCRYPT" == false ] && [ "$ASSUME_SSL" == false ]; }; then
+  if [ ! "$ASSUME_SSL" ] && [ ! "$CONFIGURE_LETSENCRYPT" ]; then
     systemctl restart nginx
```

Comment:
Don't we want it to be restarted even when Certbot runs or is this handled somewhere else?

Reply:
If` certbot` runs we don't need to restart, because using `--nginx` already does that.